### PR TITLE
[13.0] Add project_task_brand and sale_timesheet_brand

### PR DESCRIPTION
**Pull request by adrienpeiffer**
Project Task Brand
==================

This module adds the brand field to the tasks.

### Usage

The brand can be set from the form view of a task. The information is also displayed in the list view and available in the search.

Sale Timesheet Brand 
==================

This module allows to propagate the brand of the sale order on the tasks created with the products of type service

### Usage

When validating a sale order that contains service type products that are configured to create tasks, the brand indicated on the sale order is propagated to the task.

### CODE REVIEW

**Review by acsonefho — APPROVED** [review]

File: project_task_brand/models/project_task.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Copyright 2022 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ProjectTask(models.Model):
+
+    _inherit = "project.task"
```
Comment (sbejaoui) [inline project_task_brand/models/project_task.py]:
```suggestion
    _name = "project.task"
    _inherit = ["project.task", "res.brand.mixin"]
```

File: project_task_brand/models/project_task.py [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+    brand_id = fields.Many2one(comodel_name="res.brand", string="Brand")
```
Comment (sbejaoui) [inline project_task_brand/models/project_task.py]:
```suggestion
```

**Review by sbejaoui — CHANGES_REQUESTED** [review]

**Comment by sbejaoui** [comment]
please don't change pre-commit config (if necessary, do it in a dedicated PR ) and revert non-related updates

**Review by sbejaoui — APPROVED** [review]

**Comment by sbejaoui** [comment]
travis is red, please run pre-commit

**Comment by adrienpeiffer** [comment]
Depends on #125

**Comment by sbejaoui** [comment]
can you rebase please, #125 is merged now

**Comment by adrienpeiffer** [comment]
@sbejaoui Done !

**Pull request merged**